print/src/main.rs

```diff
@@ -92,15 +100,9 @@ fn main() -> anyhow::Result<()> {
 }
 
 // the string reported by the `--version` flag
-fn version() -> &'static str {
+fn print_version() -> Result<(), anyhow::Error> {
     // version from Cargo.toml e.g. "0.1.4"
-    let mut output = env!("CARGO_PKG_VERSION").to_string();
-
-    output.push_str("\nsupported defmt version: ");
-    output.push_str(defmt_decoder::DEFMT_VERSION);
-
-    // leak (!) heap memory to create a `&'static str` value. `String` won't work due to how
-    // structopt uses the clap API
-    // (this is only called once so it's not that bad)
-    Box::leak(Box::<str>::from(output))
+    println!("{}", env!("CARGO_PKG_VERSION").to_string());
```

Comment:
could you include the tool name here as well? so it prints `defmt-print 0.1.0` on the first line.

Reply:
Sure, I will upload a change for it.